inc/spbc-settings.php

```diff
@@ -4815,6 +4825,59 @@ function spbc__key_not_valid_tab_html()
            . '</a>.</h3></div>';
 }
 
+/**
+ * Generate HTML code for accordions to suggest user manual audit services.
+ * @param $for string destination accordion name
+ * @return string html
+ */
+function spbc__manual_cure_service_suggest_html($for)
+{
+    if ($for === 'critical') {
+        $text = __('With a high degree of probability, your site has been infected. If you need professional help 
+        from security specialists, feel free to order', 'security-malware-firewall');
+        $button_id = 'spbct_request_cure';
+        $classes = 'notice notice-warning';
+        $show_exclaim_triangle = true;
+    } elseif ($for === 'unknown') {
+        $text = __('If you are not sure about the results or looking for professional help from security specialists,
+         feel free to order', 'security-malware-firewall');
+        $button_id = 'spbct_request_audit';
+        $classes = 'notice notice-info';
+        $show_exclaim_triangle = false;
+    } else {
+        return '';
+    }
+
+    $cure_request_url = 'https://cleantalk.org/my/support/open';
+    $help_href_text = __('the Professional Security Audit service', 'security-malware-firewall');
+    $button_text = __('Request Professional Security Audit', 'security-malware-firewall');
+    $href = '<a target="_blank" href="https://cleantalk.org/help/manual-cure-of-malware">' . $help_href_text . '</a>';
+
+    $out = '<div id="spbc_notice_cloud_analysis_feedback" class="' . $classes . '">';
+    $out .= '<p>';
+    // show triangle
+    $out .= $show_exclaim_triangle
+        ? '<img src="http://wp60.loc/wp-content/plugins/security-malware-firewall/images/att_triangle.png" alt="attention" style="margin-bottom:-1px">&nbsp'
```

Comment:
It is necessary to remove the local domain.

Reply:
Done, thanks.